services/businessconfig/src/main/modeling/swagger.yaml

```
@@ -852,3 +890,64 @@
+      filters:
+        description: process monitoring filters
+        properties:
+          tags:
+            description: Array of filter tags for process monitoring
+            type: array
+            items:
+              $ref: '#/definitions/ProcessMonitoringFilterTag'
+          pageSize:
+            description: Number of items to display per page in the process monitoring view
+            type: integer
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)